docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,75 @@
+@startuml
+!include style.puml
+
+skinparam sequence {
+    ParticipantFontSize 25
+    ArrowFontSize 25
+}
+
+box Logic LOGIC_COLOR_T1
+participant LogicManager LOGIC_COLOR
+participant AddressBookParser LOGIC_COLOR
+participant FindAnyCommandParser LOGIC_COLOR
+participant "f:FindAnyCommand" as FindAnyCommand LOGIC_COLOR
+participant CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant Model MODEL_COLOR
+participant "p:NameContainsKeywordPredicate" as namePredicate MODEL_COLOR
+participant "c:CombinedPredicate" as combinedPredicate MODEL_COLOR
+end box
+
+-> LogicManager : execute("findAny n/alex g/m")
```

Comment:
is `"findAny n/alex g/m"` supposed to have `g/m`?

Reply:
woops